### DISCUSSION TITLE
Github page not found

### DISCUSSION THREAD

**Discussion by Dantedavidson** [community/community#21839] [Other Feature Feedback, Questions, & Ideas]
<p>I keep getting a 404 message when i try to deploy my site. When i had a read me file it would deploy that and now that only have the index.html file at the root im getting this 404 message.</p>
<p><a href="https://github.com/Dantedavidson/etch" rel="nofollow noopener">https://github.com/Dantedavidson/etch</a></p>
<p><a href="https://dantedavidson.github.io/etch/" rel="nofollow noopener">https://dantedavidson.github.io/etch/</a></p>

**Comment by Gisgar3** [comment, marked as answer]
<p>Looking at your repository, it seems that the <em>index.html</em> file is actually named <em>index .html</em>, containing a space. I would believe this would create an issue with deploying your repository to  <strong>GitHub Pages</strong>. Try renaming the file and giving time for the repository to update. This may fix your problem!</p>
<p>Hope this helps!</p>

**Comment by Dantedavidson** [comment]
<p>Cant believe i missed that, I checked it so many times for a spelling mistake. Thanks!</p>

**Comment by Gisgar3** [comment]
<p>Haha it happens! Glad to hear your problem is fixed!</p>

**Comment by Alphonso328** [comment]
<p>i am having a similar issue when i use the rc folder for the gh-pages sorce it loads however, i am using angular when i use the dist folder the page comes up blank or 404.</p>
